### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Svitlana-Yudina**

### CODE REVIEW

File: src/controllers/expensesControllers.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const userSevises = require('../servises/userServises');
```
Comment (mitattt) [inline src/controllers/expensesControllers.js:3]:
```suggestion

```

u did it in userControllers file

File: src/servises/expensesServises.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+function getNewId() {
+  const maxId = Math.max(...expenses.map(el => el.id));
+
+  if (maxId) {
+    return 1;
+  }
+
+  return maxId + 1;
+}
```
Comment (mitattt) [inline src/servises/expensesServises.js:13]: is better to use ternary operator here

File: src/servises/expensesServises.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  if (from) {
+    expensesToShow = expensesToShow
+      .filter(exp => exp.spentAt >= from);
+  }
+
+  if (to) {
+    expensesToShow = expensesToShow
+      .filter(exp => exp.spentAt <= to);
+  }
```
Comment (mitattt) [inline src/servises/expensesServises.js:45]: u could combine 2 cases in 1

File: src/servises/userServises.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+function getNewId(entity) {
+  const maxId = Math.max(...entity.map(el => el.id));
+
+  if (maxId) {
+    return 1;
+  }
+
+  return maxId + 1;
+}
+
```
Comment (mitattt) [inline src/servises/userServises.js:18]: ternary

**Review by mitattt — APPROVED** [review]
pretty good job 👍

File: src/servises/expensesServises.js (hunk 2 above)
Comment (Svitlana-Yudina) [reply src/servises/expensesServises.js:13]: I've tried... I don't understand why it doesn't work(((

File: src/controllers/expensesControllers.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+const expensesSevises = require('../servises/expensesServises');
```
Comment (illiarerun) [inline src/controllers/expensesControllers.js:4]:
```suggestion
const expensesServices = require('../services/expensesServices');
```

File: src/controllers/userControllers.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const userSevises = require('../servises/userServises');
```
Comment (illiarerun) [inline src/controllers/userControllers.js:3]: same here

File: src/servises/expensesServises.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  userId,
+  category,
+  from,
+  to,
+) {
+  let expensesToShow = expenses;
+
+  if (userId) {
+    expensesToShow = [expensesToShow
+      .find(exp => exp.userId === Number(userId))];
```
Comment (illiarerun) [inline src/servises/expensesServises.js:29]: hmm
```suggestion
    expensesToShow = expensesToShow
      .find(exp => exp.userId === Number(userId));
```

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**